RELEASING.md

```diff
@@ -0,0 +1,22 @@
+Release Guide
+-------------
+
+1. Ensure that pulpcore of the corresponding version is available on PyPI.
+1. Generate changelog (`towncrier --yes --version 3.8.0`).
+1. Update version in the `galaxy.yml` and in the `roles/pulp_common/defaults/main.yml`.
+1. Update references to current and previous versions in `docs/index.md`
```

Comment:
We do post-releases in pulp_installer as needed. The format is like "3.7.0" -> "3.7.0-1" -> "3.7.0-2" -> "3.8.0". It's why we do not use bumpversion. We should mention this in the examples..